WireAPI/Tests/WireAPITests/Helpers/String+Error.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+
+extension String: Error {}
```

Comment:
Hm is this a solution for `WireAPITests` in general? not sure if Apple has intended error handling in this way..

Reply:
Often test helpers can throw but the errors are just informative rather than something to handle in the calling code. With this extension you can just throw the error message rather than wrap it in some container struct.